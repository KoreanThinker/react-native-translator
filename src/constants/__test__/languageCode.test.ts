import LANGUAGE_CODES from '../languageCode';

test('snapshot', () => {
  expect(LANGUAGE_CODES).toMatchSnapshot();
});
