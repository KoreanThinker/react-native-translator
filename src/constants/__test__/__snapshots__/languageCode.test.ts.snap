// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
{
  "google": [
    "af",
    "ga",
    "sq",
    "it",
    "ar",
    "ja",
    "az",
    "kn",
    "eu",
    "ko",
    "bn",
    "la",
    "be",
    "lv",
    "bg",
    "lt",
    "ca",
    "mk",
    "zh-CN",
    "ms",
    "zh-TW",
    "mt",
    "hr",
    "no",
    "cs",
    "fa",
    "da",
    "pl",
    "nl",
    "pt",
    "en",
    "ro",
    "eo",
    "ru",
    "et",
    "sr",
    "tl",
    "sk",
    "fi",
    "sl",
    "fr",
    "es",
    "gl",
    "sw",
    "ka",
    "sv",
    "de",
    "ta",
    "el",
    "te",
    "gu",
    "th",
    "ht",
    "tr",
    "iw",
    "uk",
    "hi",
    "ur",
    "hu",
    "vi",
    "is",
    "cy",
    "id",
    "yi",
  ],
  "kakao": [
    "kr",
    "en",
    "jp",
    "cn",
    "vi",
    "id",
    "ar",
    "bn",
    "de",
    "es",
    "fr",
    "hi",
    "it",
    "nl",
    "pt",
    "ru",
    "th",
    "tr",
  ],
  "papago": [
    "ko",
    "en",
    "ja",
    "zh-CN",
    "zh-TW",
    "vi",
    "id",
    "th",
    "de",
    "ru",
    "es",
    "it",
    "fr",
  ],
}
`;
